### PR TITLE
CDPSDX-3537: remove invalid state update from database backup

### PR DESCRIPTION
**Pull request by RokolyA94**
JIRA: https://jira.cloudera.com/browse/CDPSDX-3537

Remove misleading status updates from, database backup.

Tested locally:
- resize test passed
- backup test passed

### CODE REVIEW

**Review by kkalvagadda1 — COMMENTED** [review]
Did you validate below with your code changes?

1. Make sure the event log saying database backup is complete is seen.
2. Make sure that the Status reason in the UI is still "Datalake backup in progress" when the database backup is complete.

**Comment by lnardai** [comment]
Please resolve the conflicts!

**Comment by lnardai** [comment]
Please resolve conflicts.

File: datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java [hunk 1]
```diff
@@ -229,6 +231,7 @@ protected void doExecute(SdxContext context, SdxEvent payload, Map<Object, Objec
                 String operationId = (String) variables.get(OPERATION_ID);
                 String backupId = (String) variables.get(BACKUP_ID);
                 SdxCluster sdxCluster = sdxService.getById(payload.getResourceId());
+                eventSenderService.sendEventAndNotification(sdxCluster, ThreadBasedUserCrnProvider.getUserCrn(), DATALAKE_DATABASE_BACKUP_FINISHED);
```
Comment (lnardai) [inline datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java:234]: Are you sure this is the finished state not the in progress?

Comment (lnardai) [reply datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupActions.java:234]: Nevermind, the DB backup is finished.

**Review by lnardai — APPROVED** [review]

**Review by biharitomi — APPROVED** [review]

**Pull request merged**